renderdoc/data/glsl/mesh.geom

```diff
@@ -22,6 +22,12 @@
  * THE SOFTWARE.
  ******************************************************************************/
 
+#ifdef OPENGL_ES
+//#extension GL_EXT_geometry_shader : enable
+//#extension GL_OES_geometry_shader : enable
+//#extension GL_EXT_geometry_point_size : enable
```

Comment:
And what do you think about `#extension_gles` and `#extension_nongles`? With the latter version we don't have to repeat the same extensions for OpenGL and Vulkan.